### PR TITLE
chore(docs): update cross-seed tutorial

### PR DESCRIPTION
**Pull request by martylukyy**
This PR updates the cross-seed setup tutorial on the docs.
Please thoroughly review the iptables section and the filter creation part
and feel free to make any changes.
I don't have any strong feeling about the changes I did.

EDIT:
The tutorial uses the apikey query param instead of the X-Api-Key HTTP header..
If there is a reason why it's better to use the header feel free to elaborate!

### CODE REVIEW

File: docs/3rd-party-tools.md [hunk 1]
```diff
@@ -118,14 +123,25 @@ sudo journalctl -u cross-seed # view the logs
...
-1. Create a filter and name it eg. `cross-seed`.
-2. Select all the indexers you want to use, preferably all of them.
-3. Set a really high `priority` to make sure it's always higher than your other filters.
-4. Go to the `External` tab, enable the `Webhook` switch, and add the following below it:
+1. Get your API key with the following command:  
+```
+cross-seed api-key
+```
+&nbsp&nbsp&nbsp&nbsp&nbsp Keep this key at hand since we will need it at step 5 later on.  
+&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp In the rest of this tutorial, we will refer to this as `YOUR_API_KEY`.
```
[the remark this answers is not in the document]
Comment (martylukyy) [reply docs/3rd-party-tools.md:131]: hmm.. i don't wanna disrupt the numbered list by a info box...
i agree it's an ugly solution but personally i don't know what would be best..

File: docs/3rd-party-tools.md [hunk 2]
```diff
@@ -118,14 +122,26 @@ sudo journalctl -u cross-seed # view the logs
...
-   Expected http status: `200`  
-   Data (JSON):
+
+2. Create a filter and name it eg. `cross-seed`.
+3. Select all the indexers you want to use, preferably all of them.
+4. Set a really high `priority` to make sure it's always higher than your other filters.
+5. Go to the `External` tab, enable the `Webhook` switch, and add the following below it:
+
+
+   - Host: `http://localhost:2468/api/announce?apikey=YOUR_API_KEY`
```
Comment (s0up4200) [inline docs/3rd-party-tools.md:140]: z made sure it accepts the `x-api-key` header, so maybe we should recommend that approach?

Comment (martylukyy) [reply docs/3rd-party-tools.md:140]: Sure can you do the changes or do you want me to do them?
Note: https://github.com/cross-seed/cross-seed/pull/543 there is a bug with the URL param too currently

**Review by s0up4200 — APPROVED** [review]
Made some adjustments.

File: docs/3rd-party-tools.md (hunk 2 above)
Comment (zakkarry) [reply docs/3rd-party-tools.md]: > Sure can you do the changes or do you want me to do them? Note: [cross-seed/cross-seed#543](https://github.com/cross-seed/cross-seed/pull/543) there is a bug with the URL param too currently

I just merged the PR to master, should be pushed to :latest soon.

**Pull request merged**